components/consumers/elasticsearch/main.go

```diff
@@ -10,12 +10,15 @@ import (
 	"strings"
 	"time"
 
+	"github.com/go-errors/errors"
+
 	v1 "github.com/ocurity/dracon/api/proto/v1"
 	"github.com/ocurity/dracon/components/consumers"
 	"github.com/ocurity/dracon/pkg/enumtransformers"
 	"github.com/ocurity/dracon/pkg/templating"
 
 	elasticsearch "github.com/elastic/go-elasticsearch/v8"
+	"github.com/elastic/go-elasticsearch/v8/esapi"
```

Comment:
nit: these imports should be grouped with the `go-errors` one.

(also, this is something that should be done automatically by the formatter but it's not being picked up)